game/platform.cc

```diff
@@ -17,7 +17,7 @@ std::uint16_t Platform::shortcutModifier(bool eitherSide) {
 
 Platform::Platform() {
 	#if (BOOST_OS_WINDOWS)
-	_putenv_s("FONTCONFIG_PATH",".\\etc\\");
+	_putenv_s("FONTCONFIG_PATH",".\\");
```

Comment:
To what is this relative? The executable? AppData? ProgramData?

Reply:
Just checked this. For vcpkg the path should be: `<vcpkg_root>/installed/<triplet>/etc`  not sure for mingw64

Reply:
Relative to the executable. I just put it in the same dir because it doesn't really make much sense to have an etc dir just to put a fonts dir under which to put the config files IMO.